python_modules/tracereplay/tracereplay/cinterface/tracereplay.c

```diff
@@ -1060,76 +1066,134 @@ static PyObject* tracereplay_populate_llseek_result(PyObject* self,
 }
 
 
-/* void print_sigset_t(sigset_t *set) */
-/* { */
-/*   int i; */
+struct kernel_sigaction {
+  __sighandler_t k_sa_handler;
+  unsigned int sa_flags;
+  void* sa_restorer;
+  sigset_t sa_mask;
+};
+
 
-/*   i = SIGRTMAX; */
-/*   do { */
-/*     int x = 0; */
-/*     i -= 4; */
-/*     if (sigismember(set, i+1)) x |= 1; */
-/*     if (sigismember(set, i+2)) x |= 2; */
-/*     if (sigismember(set, i+3)) x |= 4; */
-/*     if (sigismember(set, i+4)) x |= 8; */
-/*     printf("%x", x); */
-/*   } while (i >= 4); */
-/*   printf("\n"); */
-/* } */
+static PyObject* tracereplay_build_rt_sigaction_struct(PyObject* self,
+					   PyObject* args) {
+
+  // this is commented out because it provides no useful information and so its used / purpose needs to be reevaluated
```

Comment:
If we want to keep this code I'd save it out separately.  We don't want to clutter up master